content/docs/2.6/scalers/datadog.md

```diff
@@ -99,3 +107,25 @@ spec:
     authenticationRef:
       name: keda-trigger-auth-datadog-secret
 ```
+
+## Polling intervals and Datadog rate limiting
```

Comment:
Is this applicable to the released version of KEDA as well? ie without https://github.com/kedacore/keda/pull/2694